packages/graphql-connection-transformer/src/resources.ts

```diff
@@ -189,4 +193,217 @@ export class ResourceFactory {
             )
         }).dependsOn(ResourceConstants.RESOURCES.GraphQLSchemaLogicalID)
     }
+
+    // Resources for new way to parameterize @connection
+
+    /**
+     * Create a get item resolver for singular connections.
+     * @param type The parent type name.
+     * @param field The connection field name.
+     * @param relatedType The name of the related type to fetch from.
+     * @param connectionAttributes The names of the underlying attributes containing the fields to query by.
+     * @param keySchema Key schema of the index or table being queried.
+     */
+    public makeGetItemNewConnectionResolver(type: string,
```

Comment:
Naming came up in other comments by @yuth, perhaps using the term ParameterizedConnection would be a good name for this?